testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/AppSettings.kt

```diff
@@ -174,6 +188,13 @@ private class Settings(context: Context) {
         get() = prefs.getString(URL_PARAMETERS, null)
         set(urlParameters) = prefs.edit().putString(URL_PARAMETERS, urlParameters).apply()
 
+    var miniAppAnalyticsConfigs: List<MiniAppAnalyticsConfig>?
```

Comment:
are we using this value somewhere? or, is there any plan in future e.g. using it in an UI.

Reply:
@khairul-alam-licon yes we are not using it now, i think it will be useful for future to implement UI, now its just hard coded so i thought if we add ui later it will be useful to use this cache

Reply:
I got it, we can add it in the same PR with UI implementation, so we can make things clean as possible.